Docker/Production/entando-full-stack/postgresql/settings.xml

```diff
@@ -0,0 +1,229 @@
+                    </snapshots>
+                </repository>
+            </repositories>
+            <pluginRepositories>
+                <pluginRepository>
+                    <id>jboss-community-plugin-repository</id>
+                    <url>https://repository.jboss.org/nexus/content/groups/public/</url>
+                    <releases>
+                        <enabled>true</enabled>
+                    </releases>
+                    <snapshots>
+                        <enabled>false</enabled>
+                    </snapshots>
+                </pluginRepository>
+            </pluginRepositories>
+        </profile>
+
+        <!-- JBoss Community Maven repository (HTTP version, disabled by default) -->
+        <profile>
+            <id>jboss-community-repository-insecure</id>
+            <repositories>
+                <repository>
+                    <id>jboss-community-repository</id>
+                    <url>http://repository.jboss.org/nexus/content/groups/public/</url>
+                    <releases>
+                        <enabled>true</enabled>
+                    </releases>
+                    <snapshots>
+                        <enabled>false</enabled>
+                    </snapshots>
+                </repository>
+            </repositories>
+            <pluginRepositories>
+                <pluginRepository>
+                    <id>jboss-community-plugin-repository</id>
+                    <url>http://repository.jboss.org/nexus/content/groups/public/</url>
+                    <releases>
+                        <enabled>true</enabled>
+                    </releases>
+                    <snapshots>
+                        <enabled>false</enabled>
+                    </snapshots>
+                </pluginRepository>
+            </pluginRepositories>
+        </profile>
+
+        <!-- Override the repository "central" from the Maven Super POM, to set HTTPS by default -->
+        <profile>
+            <id>securecentral</id>
+            <repositories>
+                <repository>
+                    <id>central</id>
+                    <url>https://repo1.maven.org/maven2</url>
+                    <releases>
+                        <enabled>true</enabled>
+                    </releases>
+                </repository>
+            </repositories>
+            <pluginRepositories>
+                <pluginRepository>
+                    <id>central</id>
+                    <url>https://repo1.maven.org/maven2</url>
+                    <releases>
+                        <enabled>true</enabled>
+                    </releases>
+                    <snapshots>
+                        <enabled>false</enabled>
+                    </snapshots>
+                </pluginRepository>
+            </pluginRepositories>
+        </profile>
+
+        <!-- central repositories via HTTP. Disabled by default. -->
+        <profile>
+            <id>insecurecentral</id>
```

Comment:
Do we need this one given the https option above?

Reply:
TBH this was copied from the official EAP s2I settings file. I tend to copy this file across to all of the images which is the real problem here - I know Git allows links ...? Some of the S2I images require all of these settings